fips_assessment.yml

```diff
@@ -0,0 +1,32 @@
+---
+- name: Create knative serving
+  kubernetes.core.k8s:
+    name: knative-serving
+    namespace: knative-serving
+    api_version: operator.knative.dev/v1beta1
+    kind: KnativeServing
+    definition:
+      metadata:
+        labels:
+          opdev: opcap
+    state: present
+- name: Get Conditions
+  kubernetes.core.k8s_info:
+    namespace: knative-serving
+    name: knative-serving
+    api_version: operator.knative.dev/v1beta1
+    kind: knativeserving
+  register: knative_serving
+  until: knative_serving.resources[0].status.conditions is defined
```

Comment:
Check that `wait_condition` from k8s_info wouldn't work for both this task and the debug that comes next?

Reply:
Yeah, I was in the middle of solving that. I'll try the wait condition.